### PR TITLE
Draft May newsletter

### LINKED ISSUE #545: Send out May newsletter
### Summary

We're due an update on the project - this is probably the last thing I need to do before the wrap up

### What needs to be done?

Draft a newsletter covering
* the move of the book repo back into the main repo
* book dash reports, building on #520 
* maybe something on the hack @sgibson91 attended
* JOSE submission
* as always; conferences attended and upcoming conferences
* any updates about project going forward

### Who can help?

Everyone suggesting additional content

### PR DESCRIPTION
**Pull request by pherterich**
### Summary

Draft of a May/June newsletter.

Fixes #545 

### List of changes proposed in this PR (pull-request)

* Draft newsletter added in communications/newsletters/ folder

### What should a reviewer concentrate their feedback on?

- [ ] Rephrase or expand where appropriate
- [ ] Is anything else missing?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.

### CODE REVIEW

**Comment by rainsworth** [comment]
Looks great @pherterich!

Should we also add that the first 4 versions are up on [zenodo](https://doi.org/10.5281/zenodo.3233853)?

File: communications/newsletters/newsletter05_May2019.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+After we announced in our last newsletter that we split off the handbook in a separate GitHub repository to the main project repository, we decided to move the book back into the main Turing Way repository. 
+We realized that there wasn't a technical reason for having the book in a separate repository and splitting discussions across two repositories is rather confusing. 
+Thus, you can find everything at https://github.com/alan-turing-institute/the-turing-way and we have now also preserved various releases of the handbook [on zenodo](https://doi.org/10.5281/zenodo.3233853).
+
+## Book dashes 
+
+We were busy organising and running two very succcessful book dashes in May. 
+Our events in Manchester and London were great opportunities to grow the Turing Way community and we were really lucky to be joined by an illustrator from [scriberia](http://www.scriberia.co.uk/) who captured our book dahes wonderfully:
+
+![](https://pbs.twimg.com/media/D6zGuPiW4AI3m8s.jpg)
```
Comment (KirstieJane) [inline communications/newsletters/newsletter05_May2019.md]: Can we add this image to the github repo and link to that?

File: communications/newsletters/newsletter05_May2019.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Sarah Gibson attended the Microsoft Reactor Research Software Engineering Hackday and led a team on BinderHub that improved [our Zero to BinderHub instructions](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md).
+
+### Where to meet us in the next few weeks
+
+* Rosie Higman will present on the Turing Way at the NOWAL Research Support meeting [in Manchester on 13 June 2019](https://www.nowal.ac.uk/research-support-open-research-exchange-experience-university-manchester).
+* Kirstie Whitaker will re-run a shortened version of our March "Boost your reproducibilty with Binder" workshop at [Carpentry Connect in Manchester on 26 June 2019](https://www.software.ac.uk/ccmcr19/programme). 
+Turing Way team member Anna Krystalli will run a ReproHack, a one day hackathon that will attempt to reproduce results in published papers.
```
Comment (KirstieJane) [inline communications/newsletters/newsletter05_May2019.md]: If this doesn't have a date then I'd recommend pulling it out....

File: communications/newsletters/newsletter05_May2019.md (hunk 1 above)
Comment (pherterich) [reply communications/newsletters/newsletter05_May2019.md]: Will do, where do figures and images live now that we moved them around a bit?

Comment (KirstieJane) [reply communications/newsletters/newsletter05_May2019.md]: Aaaah - rubbish. Make a figures folder for the newsletter?

**Comment by KirstieJane** [comment]
Merging! Can always update the link to the figure in the future if we want to!

**Pull request merged**